src/index.html

```diff
@@ -9,7 +9,7 @@
   <body>
       <div id='list-container' class='container w-25 shadow my-5'>
         <ul class="list-group join-list-bottom no-round-borders">
-          <li class="list-group-item font-bold">Today's To Do</li>
+          <li class="list-group-item font-bold">Today's To Do List</li>
           <li class='list-group-item d-flex'><input id='add-list' placeholder='Add to your list...'><button id='add-btn' type='submit' class='btn btn-secondary'>Add</button></li>
         </ul>
         <ul id='list-objects' class="list-group join-list-top join-list-bottom">
```

Comment:
#### It seems like you are either repeating a week or the code you used 7 days ago, which is complicated to add inline comment however there are issuea you need to implement:

- [ ] I would recommend to modify the implementation you are trying to use while initializing as well as populate data.
 
- [ ] An array of some simple to do tasks (array of objects) with each object containing three keys - description [string], completed [bool], index: [number].

- [ ] It has a function that iterates over the tasks array and populate HTML list item element for each task.

- [ ] Make sure data are being stored and retrieved from localStorage